packages/dx-react-grid/src/plugins/table-column-visibility.jsx

```diff
@@ -65,7 +65,8 @@ export class TableColumnVisibility extends React.PureComponent {
         <Template name="table">
           {params => (
             <TemplateConnector>
-              {({ columns }) => (columns.length === hiddenColumns.length
+              {({ tableColumns }) =>
+                (isEmptyMessageShow(tableColumns)
```

Comment:
Do we really need the separated `isEmptyMessageShow` helper? It doesn't contain any complex logic and is used only once.

Reply:
For simple testing this functionality.